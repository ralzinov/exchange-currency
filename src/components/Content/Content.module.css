.host {
    display: flex;
    flex: 1;
    width: 100%;
    max-width: 992px;
    margin: 0 auto;
    padding: 16px;
}
