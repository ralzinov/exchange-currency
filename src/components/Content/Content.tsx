import * as React from 'react';
import {ReactNode} from 'react';
import styles from './Content.module.css';

interface IContentProps {
    children: ReactNode;
}

export const Content = (props: IContentProps) => {
    return (
        <div className={styles.host}>{props.children}</div>
    );
};
