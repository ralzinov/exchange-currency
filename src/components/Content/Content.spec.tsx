import React from 'react';
import ReactDOM from 'react-dom';
import {Content} from './Content';
import {create} from 'react-test-renderer';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Content>Some content</Content>, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('should render with props and children', () => {
    const button = create(
        <Content>Some content</Content>
    );
    expect(button.toJSON()).toMatchSnapshot();
});
