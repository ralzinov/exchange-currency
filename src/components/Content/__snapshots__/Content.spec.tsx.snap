// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render with props and children 1`] = `
<div
  className="host"
>
  Some content
</div>
`;
